alot/crypto.py

```diff
@@ -196,17 +169,18 @@ def verify_detached(message, signature):
 
     :param message: the message as `str`
     :param signature: a `str` containing an OpenPGP signature
-    :returns: a list of :class:`gpgme.Signature`
+    :returns: a list of :class:`gpg.results.Signature`
     :raises: :class:`~alot.errors.GPGProblem` if the verification fails
     '''
-    message_data = StringIO(message)
-    signature_data = StringIO(signature)
-    ctx = gpgme.Context()
-
-    status = ctx.verify(signature_data, message_data, None)
-    if isinstance(status[0].status, gpgme.GpgmeError):
-        raise GPGProblem(status[0].status.message, code=status[0].status.code)
-    return status
+    ctx = gpg.core.Context()
+    try:
+        verify_results = ctx.verify(message, signature)[1]
```

Comment:
I am sorry I understand neither the docs you linked to (and `verify` is not listed there) nor the code I installed via pip or in the code I cloned (it has the verify `method` but the second element in the returned tuple is created by `self.op_verify_result()` and I can not find the definition for this anywhere. I assume it was created by some metaprogramming and wraps the C function `gpgme_op_verify_result`).